BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -336,7 +336,9 @@ public enum WarheadTypes { Standard, ContinuousRod, EMP, Nuke }
 
         public float TimeToImpact { get; set; }
 
-        public int loftState = 0;
+        public enum loftStates { Boost, Midcourse, Terminal }
```

Comment:
For consistency with other enums and user-defined types, could you capitalise the first letter of `LoftStates`?